packages/backend/src/studio-api-impl.ts

```diff
@@ -60,6 +60,20 @@ export class StudioApiImpl implements StudioAPI {
     private snippetManager: SnippetManager,
   ) {}
 
+  async requestDeleteConversation(conversationId: string): Promise<void> {
+    // Do not wait on the promise as the api would probably timeout before the user answer.
+    podmanDesktopApi.window
+      .showWarningMessage(`Are you sure you want to delete this conversation ?`, 'Confirm', 'Cancel')
```

Comment:
Not sure why it is not playground as this is what user will see in the UI

Reply:
Because currently there is no consensus on what it is, sometime it is a conversation, sometime a playground, there are two objects that should be merged. At the end, it will be conversation because what you are deleting are the messages.

The playground is a dummy object with a duplicate of the conversation id, a name and modelId, which should be on the conversation object.

Reply:
My concern is that there is no mention of conversation in UI so user will be confused

Reply:
Renamed done ✅